quasar-core/src/main/java/co/paralleluniverse/fibers/instrument/Classes.java

```diff
@@ -50,12 +53,75 @@
     static final String RUNTIME_SUSPEND_EXECUTION_NAME = "co/paralleluniverse/fibers/RuntimeSuspendExecution";
     static final String UNDECLARED_THROWABLE_NAME      = UndeclaredThrowableException.class.getName().replace('.', '/');
     static final String SUSPEND_EXECUTION_NAME         = "co/paralleluniverse/fibers/SuspendExecution";
-    
-    // computed
-    static final String SUSPENDABLE_DESC = "Lco/paralleluniverse/fibers/Suspendable;";
-    static final String DONT_INSTRUMENT_DESC = Type.getDescriptor(DontInstrument.class);
-    static final String INSTRUMENTED_DESC = "Lco/paralleluniverse/fibers/Instrumented;";
-    static final String LAMBDA_METHOD_PREFIX = "lambda$";
+
+    static final String LAMBDA_METHOD_PREFIX            = "lambda$";
+
+    static final String INSTRUMENTED_DESC               = "Lco/paralleluniverse/fibers/Instrumented;";
+
+    // CORE-21 : Provide getter and setter for annotation types.
+    static class TypeDescs {
```

Comment:
Better to refer to this as `AnnotationDescriptors` ?  "Type" is a bit generic and no need to shorten "Descriptor".